src/components/ui/card.tsx

```diff
@@ -0,0 +1,158 @@
+import { forwardRef, HTMLAttributes } from 'react';
+
+import { CalendarDays, CircleCheck } from 'lucide-react';
+
+import { cn } from '@/lib/utils';
+
+interface CardHeaderProps extends HTMLAttributes<HTMLDivElement> {
+  startDate: string;
+  endDate: string;
+}
+
+interface CardContentProps extends HTMLAttributes<HTMLDivElement> {
+  category: string;
+  tenant: string;
+  isComplete: boolean;
+}
+
+/**
+ * @example
+ * ```ts
+ * return (
+ * <Card>
+ *  <CardHeader startDate={startDate} endDate={endDate} />
+ *  <CardContent category={category} tenant={tenant} isComplete={isComplete} />
+ *  <CardFooter category={category} tenant={tenant} isComplete={isComplete} />
+ * </Card>
+ * )
+ * ```
+ */
+const Card = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  ({ className, ...props }, ref) => (
+    <div
+      ref={ref}
+      className={cn(
+        'rounded-xl bg-card text-card-foreground shadow-[0_4px_8px_0_rgb(0,0,0,0.25)]',
+        className,
+      )}
+      {...props}
+    />
+  ),
+);
+Card.displayName = 'Card';
+
+const CardHeader = forwardRef<HTMLDivElement, CardHeaderProps>(
+  ({ className, startDate, endDate, ...props }, ref) => (
+    <div
+      ref={ref}
+      className={cn(
+        'flex flex-col space-y-1.5  p-[14px] border-b-2 border-slate-300',
+        className,
+      )}
+      {...props}
+    >
+      <CardTitle>Task Assignment</CardTitle>
+      <CardDescription>
+        {startDate} - {endDate}
+      </CardDescription>
+    </div>
+  ),
+);
+CardHeader.displayName = 'CardHeader';
+
+const CardTitle = forwardRef<
+  HTMLParagraphElement,
+  HTMLAttributes<HTMLHeadingElement>
+>(({ className, ...props }, ref) => (
+  <h3
+    ref={ref}
+    className={cn('text-xl font-medium leading-none tracking-tight', className)}
+    {...props}
+  />
+));
+CardTitle.displayName = 'CardTitle';
+
+const CardDescription = forwardRef<
+  HTMLParagraphElement,
+  HTMLAttributes<HTMLParagraphElement>
+>(({ className, ...props }, ref) => (
+  <div className="flex items-center gap-1">
+    <CalendarDays className="" size={16} />
+    <p
+      ref={ref}
+      className={cn('text-base text-muted-foreground', className)}
+      {...props}
+    />
+  </div>
+));
+CardDescription.displayName = 'CardDescription';
+
+const CardContent = forwardRef<HTMLDivElement, CardContentProps>(
+  ({ className, category, tenant, isComplete, ...props }, ref) => (
+    <div
+      ref={ref}
+      className={cn('p-[14px] border-b border-slate-300', className)}
```

Comment:
It looks like the top-padding is `8px` and the bottom is `10px`.

<img width="263" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/124953279/93196dd0-02f4-48df-bb13-76f1b5e6af2e">
<img width="165" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/124953279/747cf0c4-520f-4717-9724-195164ab8a9b">